generators/common/templates/quarkus/README.md.ejs

```diff
@@ -104,19 +104,13 @@ docker-compose -f src/main/docker/keycloak.yml up
 
 The security settings in `src/main/resources/config/application.yml` are configured for this image.
 
-```yaml
-spring:
-  ...
-  security:
-    oauth2:
-      client:
-        provider:
-          oidc:
-            issuer-uri: http://localhost:9080/auth/realms/jhipster
-        registration:
-          oidc:
-            client-id: web_app
-            client-secret: web_app
+```properties
+quarkus.oidc.enabled=true
+quarkus.oidc.auth-server-url=http://localhost:9080/auth/realms/jhipster/
+quarkus.oidc.client-id=web_app
+quarkus.oidc.credentials.secret=web_app
+quarkus.oidc.authentication.scopes=profile
```

Comment:
Does quarkus send `openid` by default? This usually is something like `openid,profile,email` or it's space-delimited.

Reply:
Related: we just use the defaults for Spring Security, which I believe are `openid email profile`, but I'm not certain.